src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -1,6 +1,88 @@
-import { Typography } from '@mui/material';
-import React from 'react';
+// React Imports
+import React, { useState, useEffect } from 'react';
+// MUI Imports
+import { TextField, Button } from '@mui/material';
+import { useCivicProfile } from '@hooks';
 
-const HousingInfo = () => <Typography>Housing Info</Typography>;
+/**
+ * HousingInfo - A form to fill out with housing security info
+ *
+ * @memberof CivicProfileForms
+ * @name HousingInfo
+ * @returns {React.JSX.Element} The HousingInfo Component
+ */
+const HousingInfo = () => {
+  const { data, add, isSuccess } = useCivicProfile();
+  const [formData, setFormData] = useState({
+    lastPermanentStreet: '',
+    lastPermanentCity: '',
+    lastPermanentState: '',
+    lastPermanentZIP: ''
+  });
+
+  useEffect(() => {
+    if (isSuccess) {
+      setFormData((prevFormData) => ({ ...prevFormData, ...data }));
+    }
+  }, [isSuccess, data]);
+
+  const handleChange = (event) => {
+    const { name, value } = event.target;
+    setFormData((prevFormData) => ({ ...prevFormData, [name]: value }));
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (!isSuccess) {
+      return;
+    }
+    add(formData);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div
```

Comment:
Can switch to `<Box>` or even better - `<Stack>` which already has flex and direction of column

Reply:
Think we could have this addressed in the next version in a dedicated PR like what we're doing for PR #533 with one of the other forms.